### DISCUSSION TITLE
Don't let query builder remember mapper size change done in a stats query

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#5841] [Feature Requests]
When viewing a query from stats page, one may be inclined to resize the mapping area to make up for dialog size. This size change is remembered when query builder is opened and it looks _odd_. I don't like it. Can we disconnect the two?

In theory, adding `Hide Field Mapper` button could mitigate this, but who's to say which method of minimizing would be opted for by a given user. Regardless of that addition I still think this would be a good change. 

https://github.com/specify/specify7/assets/135047322/c9c4bd58-805e-459c-8eac-b647f26b1105

**Comment by maxpatiiuk** [comment]
Good idea. Improvements on top of what you suggested:
1. Rather than separate storage for stats page query only, just store the embedded VS non embedded mapping view height in a separate place
2. Either reduce default mapping view height, or hide it completely by default

**Comment by emenslin** [comment]
Can recreate in edge (7.9.6)
